app/api-v1/api-doc.js

```diff
@@ -144,13 +171,14 @@ const apiDoc = {
           externalId: {
             description: 'id of the build in an external system',
             allOf: [{ $ref: '#/components/schemas/OnChainLiteral' }],
+            example: 'some-external-system-id',
           },
-          parts: {
-            description: 'Parts created by this build ',
+          recipes: {
+            description: 'List of recipes describing parts to be created by this build',
             type: 'array',
             maxItems: 10,
             items: {
-              $ref: '#/components/schemas/NewPart',
```

Comment:
why is this changing?

Reply:
I was thinking since we removed `orderId` from `NewPart`, it's only `recipeId` so could just be `ObjectReference`. But kind of a pointless change, I'll change it back